templates/quickstart-confluence-master.template.yaml

```diff
@@ -731,6 +731,9 @@ Conditions:
     !Equals [!Ref DBEngine, "PostgreSQL"]
 Mappings:
   AWSInstanceType2Arch:
+    c4.large:
+      Arch: HVM64
+      Jvmheap: 4608m
```

Comment:
`c4.large` instance type has only 3.75GB of RAM - https://aws.amazon.com/ec2/instance-types/

I think we would need to have a smaller heap for it. The main question is whether to support it at all. I would say, we should drop it. Not just because it is smaller but also it is a deprecated family type.

Reply:
Sounds good. I'm a fan of removing it as we should be recommending newer generations however in our Jira hardware recommendations we are still recommending c4 family for some profiles 🤔 https://confluence.atlassian.com/enterprise/infrastructure-recommendations-for-enterprise-jira-instances-on-aws-969532459.html

Reply:
So I spoke to the DC performance team and they are a bit split on the issue. They acknowledge that the c5 hardware and hypervisor are objectively better, they had better test results on the c4 instances. There is a hypothesis it is due to the dataset, however without doing a full re-test I don't think they will want to update the hardware recommendations (and I agree). I think it's best to leave it as an option for now because I think that we should align with the hardware recommendations.

Reply:
Thanks @bpar476  for the due diligence! It looks strange to recommend the older family type and have inconsistent recommendations but we should stick with our performance team results.

Reply:
I agree. They're looking into a HWR v2 so we can revisit in future.